style.css

```diff
@@ -0,0 +1,698 @@
+/*
+black: #272a31
+orange: #ec5242
+silver: #d3d3d3
+font-family: cocogoose, lato */
```

Comment:
- [ ] Here is another commented block of code. Kindly delete it

Reply:
Alright thank you